plugins/by-name/blink-cmp/default.nix

```diff
@@ -18,30 +18,62 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
   '';
 
   settingsOptions = {
-    keymap =
-      defaultNullOpts.mkNullableWithRaw (with types; either (attrsOf anything) (enum [ false ]))
-        {
-          show = "<C-space>";
-          hide = "<C-e>";
-          accept = "<Tab>";
-          select_prev = [
-            "<Up>"
-            "<C-p>"
-          ];
-          select_next = [
-            "<Down>"
-            "<C-n>"
-          ];
-          show_documentation = "<C-space>";
-          hide_documentation = "<C-space>";
-          scroll_documentation_up = "<C-b>";
-          scroll_documentation_down = "<C-f>";
-          snippet_forward = "<Tab>";
-          snippet_backward = "<S-Tab>";
-        }
-        ''
-          Customize the keymaps for blink.
-        '';
+    keymap = defaultNullOpts.mkNullableWithRaw' {
+      type = with types; either (attrsOf anything) (enum [ false ]);
+      pluginDefault = "default";
```

Comment:
```suggestion
      pluginDefault = {
        preset = "default";
      };
```

Reply:
Ah, I see I misinterpreted the upstream docs! I think I was looking at an example instead of the default value...

Luckily this is _effectively_ the same, but your original `default = 'default'` was actually more correct. Apologies!

Reply:
Don't worry, it does look better with the way you suggested and it also works. Appreciate the follow up!